src/explore-education-statistics-admin/src/pages/publication/__data__/testReleaseSeries.ts

```diff
@@ -6,7 +6,7 @@ export const testReleaseSeries: ReleaseSeriesTableEntry[] = [
     id: 'ees-release-3',
     isLegacyLink: false,
     description: 'EES release 3',
-    releaseId: 'release-id',
+    releaseId: 'release-id', // TODO rename to releaseVersionId
```

Comment:
Why not change this one in this PR? Also it would be good to have a ticket number on the TODOs to refer to.